File: Python/1018.py
n = int(input())

x_100 = n//(100)
r_100 = n%(100)

x_50 = r_100//(50)
r_50 = r_100%(50)

x_20 = r_50//(20)
r_20 = r_50%(20)

x_10 = r_20//(10)
r_10 = r_20%(10)

x_5 = r_10//(5)
r_5 = r_10%(5)

x_2 = r_5//(2)
r_2 = r_5%(2)

x_1 = r_2//(1)

print(n)
print('{} nota(s) de R$ 100,00'.format(x_100))
print('{} nota(s) de R$ 50,00'.format(x_50))
print('{} nota(s) de R$ 20,00'.format(x_20))
print('{} nota(s) de R$ 10,00'.format(x_10))
print('{} nota(s) de R$ 5,00'.format(x_5))
print('{} nota(s) de R$ 2,00'.format(x_2))
print('{} nota(s) de R$ 1,00'.format(x_1))

File: JavaScript/1018.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var n = parseInt(lines.shift());

var r_100 = n%(100)
var x_100 = Math.floor(n/(100))

var x_50 = Math.floor(r_100/(50))
var r_50 = r_100%(50)

var x_20 = Math.floor(r_50/(20))
var r_20 = r_50%(20)

var x_10 = Math.floor(r_20/(10))
var r_10 = r_20%(10)

var x_5 = Math.floor(r_10/(5))
var r_5 = r_10%(5)

var x_2 = Math.floor(r_5/(2))
var r_2 = r_5%(2)

var x_1 = Math.floor(r_2/(1))

console.log(n)
console.log(x_100 + ' nota(s) de R$ 100,00')
console.log(x_50 + ' nota(s) de R$ 50,00')
console.log(x_20 + ' nota(s) de R$ 20,00')
console.log(x_10 + ' nota(s) de R$ 10,00')
console.log(x_5 + ' nota(s) de R$ 5,00')
console.log(x_2 + ' nota(s) de R$ 2,00')
console.log(x_1 + ' nota(s) de R$ 1,00')